src/Cmd.ml

```diff
@@ -45,23 +45,30 @@ let make ?(env = Process.env) ~cmd () =
       | None -> Error {j| Command "$cmd" not found |j}
       | Some cmd -> Ok { cmd; env } )
 
-let output ~args ?cwd ?stdin { cmd; env } =
-  (* TODO use ChildProcess.spawn to get rid of this pointless concatenation *)
-  let shellString =
-    Js.Array.joinWith " " (Js.Array.concat args [| Path.toString cmd |])
+let formatArgs args =
```

Comment:
We only use this function for writing log entries, right?

In that case, I wonder if we should just write the args as a json list and save ourselves the hassle. I know this effort at quoting is going to fail in some edge case regardless?

Reply:
Yes, the quoting is only for logs. I will just use the json list then.